content-src/experiments/side-view.yaml

```diff
@@ -0,0 +1,84 @@
+id: 16
+title: 'Side View'
+slug: side-view
+is_featured: false
+platforms: ['addon']
+min_release: 60
```

Comment:
Is the min-version 57 or 60?
The side-view manifest.json seems to have a `strict_min_version` of "57.0a1" 
https://github.com/mozilla/side-view/blob/d99f76322201e402d0b3222eb03779dd34499204/addon/manifest.json.tmpl#L15